### ISSUE TITLE
Provide accessible name property to side navigation item and side navigation tags

### ISSUE THREAD

**Issue by samanr** [SAP/ui5-webcomponents#5034]
### **Feature Request Description**
As a user I would require my application to be accessible via screen readers operating on Windows and MacOs machines.
We have a requirement which is, that any side panel menu item to be read with it position in the side panel menu.
For ex:  [1 of 4, 2of 4 and so on]
We are using `SideNavigationItem` and `SideNavigationSubItem` to implement a side panel. 
There is no accessibleName property available for these components and by default these items are read as a tree item.

### **Proposed Solution**
Proposed solution would be that library provide accessibility out of the box such that it reads the label of the nav item and its position in the list of nav items

### **Proposed Alternatives**
 User should be able to add a custom label/accessibility property to obtain the same result.

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

A clear and concise description of the impact/urgency of the required feature.

### **Stakeholder Info** (if applicable)

- Organization: {...}
- Business impact: {...}

**Comment by stbodurov** [comment]
Hello @SAP/ui5-webcomponents-topic-rd,

Please take a look at this accessibility feature request regarding accessibleName in Side Navigation.

Regards,
Stanislav

**Comment by TeodorTaushanov** [comment]
This issue is answered here:
https://github.com/SAP/ui5-webcomponents/issues/5052
